exercises/hello-world/HINT.md

```diff
@@ -0,0 +1,4 @@
+The starter implementation for this exercise uses [UnsupportedOperationException](http://docs.oracle.com/javase/8/docs/api/?java/lang/UnsupportedOperationException.html). 
```

Comment:
Maybe we can add a little tidbit here on `UnsupportedOperationException` specifically, since any exception would cause a test failure with a given method. Perhaps something like "This exception is commonly thrown to alert developers that a method has not yet been implemented."